boards/arm/nrf5340_pca10095/nrf5340_pca10095_cpu1.yaml

```diff
@@ -0,0 +1,13 @@
+identifier: nrf5340_pca10095_cpu1
+name: nRF5340-PCA10095-network-MCU
+type: mcu
+arch: arm
+toolchain:
+  - gnuarmemb
+  - xtools
+  - zephyr
+ram: 64
+flash: 256
+supported:
+  - i2c
```

Comment:
No I2C instance is enabled for this board, so this entry should be removed.

Reply:
removing, thanks.